src/services/geolocationServices.ts

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import { GeolocationPosition } from '../common/model';
+import { IpInfoData } from './../common/model';
+import * as Constants from '../common/constants';
+
+// Geolocation PositionOptions properties
+// Browser geolocation timeout in milliseconds
+export const GEOLOCATION_TIMEOUT = 5000;
+// Max age for last geolocation attempt in milliseconds
+export const GEOLOCATION_MAX_AGE = 30000;
```

Comment:
Are these moved from the `timeout` val in `remoteData/asyncActionCreator` that still exists?